app/routes.js

```diff
@@ -189,11 +189,33 @@ router.get('/assessment-type-answer', function (req, res) {
   let assessmentMarkingRegex = new RegExp(/Marking assessments/i)
   let assessmentModeratingRegex = new RegExp(/Moderating assessments/i)
   let assessmentDesigningRegex = new RegExp(/Designing and setting assessments/i)
+  let assessmentSeniorRegex = new RegExp(/Designing and setting assessments/i)
 
   let isMarking = assessmentMarkingRegex.test(assessmentExpertise)
   let isModerating = assessmentModeratingRegex.test(assessmentExpertise)
   let isDesigning = assessmentDesigningRegex.test(assessmentExpertise)
 
+  if (isMarking == true) {
```

Comment:
add above this for Reviewing: 

`let isReviewing = assessmentReviewingRegex.test(assessmentExpertise)`